u2-routing-controllers/features/rgbControllers.js

```diff
@@ -0,0 +1,19 @@
+const getRgbByQuery = (req, res) => {
+  console.log(req.query)
+  const red = parseInt(req.query.r || '0')
+  const green = parseInt(req.query.g || '0')
+  const blue = parseInt(req.query.b || '0')
+  const average = Math.round((red + green + blue) / 3)
+
+  console.log({ red, green, blue, average })
+
+  res.send(`<html><body style="background-color: rgb(${red}, ${green}, ${blue});color: ${
```

Comment:
:bulb: It might be better to extract background & foreground colours to variables, so this code is simpler & more readable:
```js
const backgroundColor = ...
const color = ...
...
style="background-color: ${backgroundColor}; color: ${color};"
```